operators/community-windows-machine-config-operator/5.1.2/manifests/community-windows-machine-config-operator.5.1.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,349 @@
+                    command:
+                      - windows-machine-config-operator
+                    env:
+                      - name: WATCH_NAMESPACE
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.annotations['olm.targetNamespaces']
+                      - name: POD_NAME
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.name
+                      - name: OPERATOR_NAME
+                        value: community-windows-machine-config-operator
+                    image: quay.io/openshift-windows/community-windows-machine-config-operator:community-4.10-749c59d
+                    imagePullPolicy: IfNotPresent
+                    name: manager
+                    resources: {}
+                hostNetwork: true
+                nodeSelector:
+                  node-role.kubernetes.io/master: ""
+                priorityClassName: system-cluster-critical
+                serviceAccountName: windows-machine-config-operator
+                terminationGracePeriodSeconds: 10
+                tolerations:
+                  - effect: NoSchedule
+                    key: node-role.kubernetes.io/master
+                    operator: Exists
+                  - effect: NoExecute
+                    key: node.kubernetes.io/unreachable
+                    operator: Exists
+                    tolerationSeconds: 120
+                  - effect: NoExecute
+                    key: node.kubernetes.io/not-ready
+                    operator: Exists
+                    tolerationSeconds: 120
+      permissions:
+        - rules:
+            - apiGroups:
+                - ""
+              resources:
+                - configmaps
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - update
+                - patch
+                - delete
+            - apiGroups:
+                - coordination.k8s.io
+              resources:
+                - leases
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - update
+                - patch
+                - delete
+            - apiGroups:
+                - ""
+              resources:
+                - events
+              verbs:
+                - create
+                - patch
+          serviceAccountName: windows-machine-config-operator
+    strategy: deployment
+  installModes:
+    - supported: true
+      type: OwnNamespace
+    - supported: false
+      type: SingleNamespace
+    - supported: false
+      type: MultiNamespace
+    - supported: false
+      type: AllNamespaces
+  keywords:
+    - windows
+  maintainers:
+    - email: team-winc@redhat.com
+      name: Red Hat, Windows Container Support for OpenShift
+  maturity: preview
+  minKubeVersion: 1.24.0
```

Comment:
what is the minKubeVersion based here upon/generated from? Shouldn't it be 1.23 for 4.10

Reply:
The script generates the csv from master. I missed manually changing that since we are in the middle of prepping for 6.0.0.

Reply:
Maybe I misunderstood it but can we run the script from community-4.10 branch instead of master, I would also like to understand why we are not currently doing that?

Reply:
We are planning to release a community version every sprint so I thought we would base the community wmco bundle on the wmco master bundle so that we always have the most up-to-date community version available upstream, but I see your point on using community-4.10 branch instead so I can open a follow-up PR to change the script.